python/src/KernelSmoothing_doc.i.in

```diff
@@ -22,28 +22,67 @@ dimension, then the unit weight of each point is linearly affected to the vertic
 of the bin containing the point. See [wand1994]_ for the details.
 
 The boundary correction is available only in one dimension, and it is done using
-the mirroring technique. See the notes of the *setBoundingOption* method for the
+the mirroring technique. See the notes of the :meth:`setBoundingOption` method for the
 details.
 
 When applied to multivariate samples, the kernel is the kernel product of the
-univariate distribution specified in the constructor. 
+univariate distribution specified in the constructor.
 
 Examples
 --------
 Fit a distribution on data thanks to the kernel smoothing technique:
 
 >>> import openturns as ot
 >>> sample = ot.Gamma(6.0, 1.0).getSample(100)
->>> kernel = ot.KernelSmoothing()
->>> fittedDist = kernel.build(sample)
+>>> ks = ot.KernelSmoothing()
+>>> fittedDist = ks.build(sample)
+>>> print(fittedDist.getClassName())
+Distribution
+
+Get the bandwidth:
+
+>>> bandwidth = ks.getBandwidth()
+>>> print(bandwidth)
+[1.03374]
+
+The bandwidth was guessed by the :meth:`build` method.
+It could also have been provided by the user.
+
+>>> bandwidth = [0.9]
+>>> fittedDist = ks.build(sample, bandwidth)
 
 Compare the PDFs:
 
 >>> graph = fittedDist.drawPDF()
 >>> graph.add( ot.Gamma(6.0, 1.0).drawPDF())
->>> graph.setColors(['blue', 'red'])
+>>> graph.setColors(ot.Drawable_BuildDefaultPalette(2))
```

Comment:
I'd use "Drawable.BuildDefaultPalette" instead

Reply:
OK, but why does it matter?

Reply:
```suggestion
>>> graph.setColors(ot.Drawable.BuildDefaultPalette(2))
```

Reply:
To show that its a method of Drawable, the method with _ is just a construct of swig and not very pythonic

Reply:
OK, thanks. I have changed it everywhere.